spec/features/end_to_end_spec.rb

```diff
@@ -268,6 +268,7 @@
     select 'repair_type', from: 'Vocabulary'
     fill_in 'Key', with: 'key_string'
     check 'Active'
+    check 'Favorite'
```

Comment:
You are creating a new repair type as a favorite in the end-to-end spec, but can you also verify later in the spec when creating new in-house repairs that the repair type you created is first in the list?  I believe you can have Capybara select the first item from a list then save that new record.  If the repair type in your new repair record matches 'key_string', all is well.

Reply:
```ruby
    # get list of repair_types and check that favorite is first option
     repair_types = find('#in_house_repair_record_repair_type').all('option').collect(&:text)
     expect(repair_types[1..]).to start_with('updated_key_string')
```